src/tests/ftest/util/run_utils.py

```diff
@@ -36,6 +36,30 @@ def __init__(self, command, returncode, hosts, stdout, stderr, timeout):
         self.stderr = stderr
         self.timeout = timeout
 
+    def __lt__(self, other):
+        """Determine if another ResultData object is less than this one.
+
+        Args:
+            other (NodeSet): the other NodSet to compare
+
+        Returns:
+            bool: True if this object is less than the other ResultData object; False otherwise
+        """
+        if not isinstance(other, ResultData):
+            raise NotImplementedError
```

Comment:
I think the common exception here is `TypeError`. For example:
```
>>> 3 < "4"
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: '<' not supported between instances of 'int' and 'str'
```

Reply:
I like that, but I think I could make the change in a follow up PR.